compiler/rustc_lint/src/context/diagnostics.rs

```diff
@@ -114,6 +114,9 @@ pub(super) fn builtin(sess: &Session, diagnostic: BuiltinLintDiagnostics, diag:
                 diag.span_label(span, format!("the item `{ident}` is already {introduced} here"));
```

Comment:
I think we just need to say
```rust
"the item `{ident}` is already {introduced} by prelude"
```
if the `span` is dummy and cannot be meaningfully used for a label.